File: python/day-6.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
wordCounter = int(input())

for i in range(wordCounter):
    word = input()
    result = ""
    oddString = ""
    evenString = ""
    
    chars = list(word)
    
    i = 0
    for c in chars:
        
        if(i == 0 or i % 2 == 0):
            evenString = evenString + c
        else:
            oddString = oddString + c
        i = i +1
            
    result = evenString + " " + oddString
    print(result)

File: java/day-6.java
import java.io.*;
import java.util.*;

public class Solution {

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        
        Scanner scanner = new Scanner(System.in);
        
        int wordCounter = scanner.nextInt();
        ArrayList words = new ArrayList<String>();
        
        for(int i = 0; i<wordCounter; i++){
            
            String word = scanner.next();
            String result;
            String oddString = "";
            String evenString = "";
            
            char[] charArray = word.toCharArray();
            
            for(int j =0; j < charArray.length ; j++){
                
                if(j == 0 || j % 2 == 0){
                    evenString = evenString + charArray[j];
                } else {
                    oddString = oddString + charArray[j];
                }
            }
            
            result = evenString + " " + oddString;
            System.out.println(result);
        }
        
        
        scanner.close();
    }
}
